docs/stateful-set-migrate.md

```diff
@@ -0,0 +1,39 @@
+# Gitserver Stateful Set Migration `(2.10.* -> 2.11.*+)`
+
+Starting with Sourcegraph Data Center `2.11`, `gitserver`'s configuration has been migrated to use [StatefulSets](https://kubernetes.io/docs/tutorials/stateful-application/basic-stateful-set/) instead of having multiple deployments and associated services.
+
+In order be able to re-use your existing `gitserver`'s persistent volumes with [StatefulSets](https://kubernetes.io/docs/tutorials/stateful-application/basic-stateful-set/) (so that you can avoid re-cloning existing repositories), your cluster's administrator will need to run the following manual steps before upgrading to `2.11` ([click here to read more about why this is necessary](https://github.com/kubernetes/kubernetes/issues/48609#issuecomment-314066616)):
+
+## Steps
+
+_Note that these steps will cause a small amount of unavoidable downtime._
+
+_The following steps assume that you have [jq](https://stedolan.github.io/jq/) installed._
+
+1. Make sure that the persistent volumes for your existing `gitserver` deployment are marked as `retained`
```

Comment:
The command looks like it updates the PVs rather than just displaying their status, so a better sentence would be "Set the Reclaim Policy for ... to Retain."